packages/@biomejs/biome/configuration_schema.json

```diff
@@ -1182,6 +1182,26 @@
 			},
 			"additionalProperties": false
 		},
+		"ImportExtensionMapping": {
+			"type": "object",
+			"required": ["componentImportExt", "fileExt", "importExt"],
+			"properties": {
+				"componentImportExt": {
+					"description": "Extension that should be used for component file imports",
```

Comment:
what is the difference between component file and import in this context?

Reply:
Rule defines component files as ones starting with an upper-case letter - I guess it is common for React / Vue components - hence the ability to use a different `jsx` / `tsx` extension.

Reply:
This was a stop gap solution until Biome has infra to check filesystem. Bit unfortunate we need to expose that option, but not sure more general solution is possible.

Though maybe if user picks `.js` we could auto pick `.jsx` for those component imports, and likewise `.ts` -> `.tsx`? Or would that be too much magic?

Reply:
This is a more or less advanced configuration - personally I would prefer to just have an explicit configuration.